### ISSUE TITLE
Info on project status?

### ISSUE THREAD

**Issue by SiShaman** [Trivadis/plsql-cop-cli#48]
Hi there. We had a license for db*CODECOP and recently received notice that it’s no longer being developed, at least officially. I was hoping to get some clarification/confirmation since it has been a valuable tool for us for years. We can, of course, create our own extensions but I was wondering if perhaps development would continue under auspices outside of Accenture.

**Comment by PhilippSalvisberg** [comment]
> I was wondering if perhaps development would continue under auspices outside of Accenture.

Yes, we are currently working on a new product suite. Please contact us at info@grisselbav.com to learn more. Thank you very much.
